app/views/layouts/application.html.haml

```diff
@@ -0,0 +1,13 @@
+!!!
+%html
+  %head
+    %meta{:content => "text/html; charset=UTF-8", "http-equiv" => "Content-Type"}/
+    %title Craftynews
+    = csrf_meta_tags
+    = csp_meta_tag
+    -# = stylesheet_pack_tag 'application'
```

Comment:
Please remove comments (if you are not going to use this one)

Reply:
Not using it now, so it's been removed now.